CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.22.0] - 2022-10-14
+
```

Comment:
Usually we change the release version separately from the changes that go into it.  Was there something that required changing the version in this PR?
```suggestion
```

Reply:
It'd be great to have a release soon to enforce using the effective canister id for provisional_create_canister_with_cycles in our tests.

Reply:
But we could also do this in a separate MR and only bump the minor version if you prefer (this PR is a breaking change though because read_state_raw takes only two arguments now).

Reply:
We also usually release a new agent right away after a change like this.  In this PR, please don't bump the major or minor versions.

Reply:
I see. Done.